### PR TITLE
Update pytest.yml

### PR DESCRIPTION
**Pull request by shak360**

### CODE REVIEW

**Comment by shak360** [comment]
I'm trying to make `pytest --cov=lineage` ignore the figures folder but I'm not sure if this is the way to do it @aarmey .

**Comment by aarmey** [comment]
This looks right as far as I can tell...

File: .github/workflows/pytest.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
         if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
     - name: Test with pytest
       run: |
-        pytest --cov=lineage --cov-report=xml
+        pytest --cov=lineage --cov-report=xml --cov-config=.coveragerc
```
Comment (aarmey) [inline .github/workflows/pytest.yml:33]: With the way you have it, this should be `--cov-config=.github/workflows/coveragerc`

**Comment by shak360** [comment]
Cool, I think it's working as intended now.

**Pull request merged**